remind_mfa/cement/cement_mfa_system_future.py

```diff
@@ -1,48 +1,70 @@
 import numpy as np
 import flodym as fd
+from enum import Enum
 
 from remind_mfa.common.assumptions_doc import add_assumption_doc
+from remind_mfa.cement.cement_carbon_uptake_model import CementCarbonUptakeModel
+
+
+class CementMode(str, Enum):
+    base = "base"
+    carbon_flow = "carbon_flow"
 
 
 class StockDrivenCementMFASystem(fd.MFASystem):
 
+    mode: CementMode
+
     def compute(self, stock_projection: fd.FlodymArray):
         """
         Perform all computations for the MFA system.
         """
         self.compute_in_use_stock(stock_projection)
         self.compute_flows()
         self.compute_other_stocks()
+        if self.carbon_flow:
+            CementCarbonUptakeModel(mfa=self).compute_carbon_flow()
         self.check_mass_balance()
         self.check_flows(raise_error=False)
 
-    def compute_in_use_stock(self, stock_projection: fd.FlodymArray):
+    def compute_in_use_stock(self, cement_stock_projection: fd.FlodymArray):
         prm = self.parameters
         stk = self.stocks
+        cement_ratio = prm["product_cement_content"] / prm["product_density"]
+
+        stk["in_use"].stock = (
+            cement_stock_projection
+            * prm["product_material_split"]
+            * prm["product_material_application_transform"]
+            * prm["product_application_split"]
+            / cement_ratio
+        )
 
-        stk["in_use"].stock = stock_projection
+        lifetime_rel_std = 0.4
         stk["in_use"].lifetime_model.set_prms(
             mean=prm["future_use_lifetime_mean"],
-            std=0.2 * prm["future_use_lifetime_mean"],
+            std=lifetime_rel_std * prm["future_use_lifetime_mean"],
         )
         add_assumption_doc(
             type="expert guess",
-            value=0.2,
+            value=lifetime_rel_std,
             name="Standard deviation of future use lifetime",
-            description="The standard deviation of the future use lifetime is set to 20 percent of the mean.",
+            description=f"The standard deviation of the future use lifetime is set to {int(lifetime_rel_std * 100)} percent of the mean.",
         )
```

Comment:
nit-picking, but assumption_doc should go directly below definition of lifetetime_rel_std.